qdrant-landing/content/articles/qdrant-1.3.x.md

```diff
@@ -33,6 +33,26 @@ Your feedback is valuable to us, and are always tying to include some of your fe
 
 ## New features
 
+### Asychronous I/O interface
+
+Going forward, we will support the `io_uring` asychnronous interface for storage devices on Linux-based systems. Since its introduction, `io_uring` has been proven to speed up slow-disk deployments wherever the OS syscall overhead gets too high and the software becomes IO bound.
+
+<aside role="status">This experimental feature works on Linux kernels > 5.4 </aside>
+
+`io_uring` uses a ring buffer data structure to queue and manage I/O operations asynchronously, minimizing the need for context switches and reducing overhead. It supports features like submission and completion queues, event-driven I/O, and batch processing, allowing for efficient handling of I/O events and reducing the CPU overhead associated with traditional I/O.
+
+#### Enable async storage interface from the storage configuration file:
+
+```python
+storage:
+	# enable the async scorer which uses io_uring
+	async_scorer: true
+```
+You can return to the mmap based backend by either deleting the `async_scorer` entry or setting the value to `false`.
+
+This operation generates a lot of disk IO, so it is a prime candidate for possible improvements.
+Please keep in mind that this feature is experimental and that the interface may change in further versions.
+
```

Comment:
@llogiq Could we make a performance claim here and add a small table of benchmark results?